helm/grip/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
What's the purpose of the `ingress.yaml`? I see it only used in revproxy, which makes sense since it's the only external facing node. In the `values.yaml`, it seems like it is `enabled: false` anyways.

Reply:
ingress is what catches all of the inward network traffic coming into rev-proxy:
https://kubernetes.io/docs/concepts/services-networking/ingress/

It's off in this chart because it's already implemented in our rev-proxy chart:
https://stackoverflow.com/questions/72229854/helm-charts-and-ingress-resources

Reply:
Deleted in most recent commit